test/target_mvd/targets_test.exs

```diff
@@ -69,7 +69,12 @@ defmodule TargetMvd.TargetsTest do
 
     setup [:create_topic, :create_user]
 
-    @valid_attrs %{latitude: 120.5, longitude: 120.5, radius: 42, title: "some title"}
+    @valid_attrs %{
+      latitude: 120.5,
```

Comment:
is there any way of faking these values?
So the test instances are not hardcoded?
Maybe you can use [exceck](https://github.com/parroty/excheck) and do some property-based testing on this. or at least use the strategies for generating the data.

Reply:
Good call, I guess we could start doing factories. Though it might be a bit early since there aren't many places in the tests where it would be needed. I agree with doing it if it breaks the rule of three